react-frontend/src/components/Nav/routes.js

```diff
@@ -14,6 +15,9 @@ const Routes = () => {
       <Route path="/resources">
         <Resources />
       </Route>
+      <Route path="/casestudies">
```

Comment:
for consistency maybe add the - to the route, not sure what best practice is though